docs/docs-beta/src/plugins/scoutos/index.js

```diff
@@ -0,0 +1,86 @@
+// ScoutOS Docusaurus Plugin
+//
+// Injects the required HTML for the ScoutOS chat bot into all pages on Docusaurus.
+//
+// USAGE
+//
+//     Include the plugin in your `docusaurus.config.js`:
+//
+//          plugins: [
+//            require.resolve('./src/plugins/scoutos'),
+//          ],
+//
+// REFERENCE
+//
+//     https://docs.scoutos.com/docs/integrations/scout-copilot
+//     https://docusaurus.io/docs/api/plugin-methods/lifecycle-apis#injectHtmlTags
+//
+
+module.exports = function (context, options) {
+  return {
+    name: 'scoutos',
+    injectHtmlTags() {
+      return {
+        headTags: [
+          {
+            tagName: 'script',
+            attributes: {
+              type: 'text/javascript',
+            },
+            // NOTE: we load the HTML after the `DOMContenteLoader` event as we want to prevent this
+            // from being statically embedded in the HTML. One reason for this is because the
+            // custom `scout-os` element fails to build due to minification errors.
+            innerHTML: `
+              document.addEventListener('DOMContentLoaded', function() {
+                var copilot = document.createElement('co-pilot');
+                copilot.setAttribute('copilot_id', 'dagster');
+                copilot.innerHTML = \`
+                  <div
+                    slot="fab"
+                    style="
+                      background-color: #E6E6FA; /* lavender */
```

Comment:
Where did these styles come from?

Reply:
Translated the tailwind CSS from the button on the legacy docs. Was hoping to make it look cooler with the "defacto" AI logo before launch and maybe some shimmer.

Reply:
@PedramNavid this seems to work:

https://github.com/dagster-io/dagster/pull/27198/commits/075312414ec45e332bf9e7c62c18bbb37792ef6c

<img width="746" alt="image" src="https://github.com/user-attachments/assets/9567abd2-753c-4f7c-a392-454757c812e8" />

<img width="642" alt="image" src="https://github.com/user-attachments/assets/7485d3f3-b856-419f-a7a3-f30d2ba4552c" />